nasa/mmt
# MMT-3769: Fixes metadata download links, MMT-3768: Use draft's provider when saving draft

# Overview

### What is the feature?

* Fixes metadata download links (point to correct CMR, and hide links for drafts)
* Use the existing draft provider when saving the draft
* Updates metadata-preview library

### What areas of the application does this impact?

* Metadata Preview, viewing published concepts and drafts
* Saving drafts

# Testing

* Fixes metadata download links (point to correct CMR, and hide links for drafts)
  * View a published concept, the Metadata Download buttons should point to the correct CMR env
  * View a draft, no Metadata Download buttons will be present
* Use the existing draft provider when saving the draft
  * With multiple providers available to you, create a new draft, with any provider except the first in the modal dropdown
  * Make a change to the metadata and save again
  * Ensure in the URL that the concept ID remains on the original provider you selected
* Updates metadata-preview library
  * The spatial map on the collection (published and draft) correctly shows the spatial overlay above the map

# Checklist
